Tasks/AndroidSigningV3/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 175,
+        "Minor": 176,
```

Comment:
Could you please bump up a minor version in `task.lock.json` as well?

Reply:
Thanks! Updated it also

Reply:
Thanks! Updated it also